generation/classifier_free_guidance/2d_ddpm_classifier_free_guidance_tutorial.ipynb

```diff
@@ -0,0 +1,959 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d5004d2e-591c-475e-be1d-d7001faf43c1",
+   "metadata": {},
+   "source": [
+    "Copyright (c) MONAI Consortium<br>\n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");<br>\n",
+    "you may not use this file except in compliance with the License.<br>\n",
+    "You may obtain a copy of the License at<br>\n",
+    "http://www.apache.org/licenses/LICENSE-2.0<br>\n",
+    "Unless required by applicable law or agreed to in writing, software<br>\n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,<br>\n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.<br>\n",
+    "See the License for the specific language governing permissions and<br>\n",
+    "limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "63d95da6",
+   "metadata": {},
+   "source": [
+    "# Classifier-free Guidance\n",
+    "\n",
+    "This tutorial illustrates how to use MONAI for training a denoising diffusion probabilistic model (DDPM)[1] to create synthetic 2D images using the classifier-free guidance technique [2] to perform conditioning.\n",
+    "\n",
+    "\n",
+    "[1] - Ho et al. \"Denoising Diffusion Probabilistic Models\" https://arxiv.org/abs/2006.11239\n",
+    "[2] - Ho and Salimans \"Classifier-Free Diffusion Guidance\" https://arxiv.org/abs/2207.12598\n",
```

Comment:
```suggestion
    "- [1] Ho et al. \"Denoising Diffusion Probabilistic Models\" https://arxiv.org/abs/2006.11239\n",
    "- [2] Ho and Salimans \"Classifier-Free Diffusion Guidance\" https://arxiv.org/abs/2207.12598\n",
```
Slightly better layout. It would also help to explain here what classifier-free guidance is and what it does for you briefly in one sentence.